@uportal/esco-content-menu/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "start": "vue-cli-service serve",
     "prebuild": "babel node_modules/@vue/web-component-wrapper/dist/vue-wc-wrapper.js -o node_modules/@vue/web-component-wrapper/dist/vue-wc-wrapper.js",
-    "build": "vue-cli-service build --name esco --target wc 'src/components/*.vue'",
+    "build": "vue-cli-service build --name esco-content-menu --target wc src/components/*.vue",
```

Comment:
`esco-content-menu` is being built in multi-component mode.
https://cli.vuejs.org/guide/build-targets.html#bundle-that-registers-multiple-web-components

The name is used as a prefix to each component.
https://github.com/uPortal-contrib/uPortal-web-components/blob/3b522fa3c9570e43377a625f8053518cd7e048a5/%40uportal/esco-content-menu/src/components/ContentMenu.vue#L81
https://github.com/uPortal-contrib/uPortal-web-components/blob/3b522fa3c9570e43377a625f8053518cd7e048a5/%40uportal/esco-content-menu/src/components/ContentGrid.vue#L115
etc.

changing this will rename the components from
```
esco-content-menu
esco-content-grid

// to

esco-content-menu-content-menu
esco-content-menu-content-grid
````

Reply:
I initially did not want to change this. I had an error from npm on linux mint saying that esco is not a valid component name. 

npm is latest lts version

Reply:
What is the output from:

```bash
# in uPortal-web-components/@uportal/esco-content-menu
node --version
npm --version
npm list @vue/cli-service
git log --oneline | head
```

Reply:
I tested this locally
the component definitions output from the build change from:

```js
window.customElements.define('esco-action-favorites', vue_wc_wrapper_default()(external_Vue_default.a, ActionFavoritesshadow))


window.customElements.define('esco-content-favorites', vue_wc_wrapper_default()(external_Vue_default.a, ContentFavoritesshadow))


window.customElements.define('esco-content-grid', vue_wc_wrapper_default()(external_Vue_default.a, ContentGridshadow))


window.customElements.define('esco-content-grid-filter', vue_wc_wrapper_default()(external_Vue_default.a, ContentGridFiltershadow))


window.customElements.define('esco-content-menu', vue_wc_wrapper_default()(external_Vue_default.a, ContentMenushadow))


window.customElements.define('esco-content-user', vue_wc_wrapper_default()(external_Vue_default.a, ContentUsershadow))


window.customElements.define('esco-ellipsis', vue_wc_wrapper_default()(external_Vue_default.a, Ellipsisshadow))


window.customElements.define('esco-hamburger-menu', vue_wc_wrapper_default()(external_Vue_default.a, HamburgerMenushadow))


window.customElements.define('esco-header-buttons', vue_wc_wrapper_default()(external_Vue_default.a, HeaderButtonsshadow))


window.customElements.define('esco-portlet-card', vue_wc_wrapper_default()(external_Vue_default.a, PortletCardshadow))
```

to:

```js
window.customElements.define('esco-content-menu', vue_wc_wrapper_default()(external_Vue_default.a, ActionFavoritesshadow))
```

the original multi-component build is the intended output.

---

the component output of a build can be found in the `dist/esco-content-menu.js` by grepping for `customElements.define`

Reply:
Do these have to be built as a multi-component build?

Reply:
Which dist folder are you in? I am looking in the esco-content-menu dir and I cant see it after building from master on linux.

Reply:
> Do these have to be built as a multi-component build?

These do, `esco-content-menu` and `esco-content-grid` are adopted independently of each other in uPortal.

> Which dist folder are you in? I am looking in the esco-content-menu dir and I cant see it after building from master on linux.

in _@uportal/esco-content-menu/dist_

To build I'm running

```bash
# delete existing node modules and builds
rm -rf **/node_modules **/dist

# re-install node modules based of lockfile
npm ci
lerna bootstrap --ci

# rebuild all projects
npm run build

# move to build folder for esco-content-menu
cd @uportal/esco-content-menu/dist
```

Reply:
When we build on 
Mac OS 10.14.3
node 10.15.1
npm 6.4.1 
we also don't have the esco-content-menu.js file.

Reply:
correction, the dist should be *esco.js*

Reply:
OK. I am seeing the error now. I have the correct output on windows with the name change, however on linux I am getting the same as you above. Lets abandon this pull request for now to see if I can find a fix. 
Thanks